packages/core/security/core-security-common/src/authentication/authenticated_user.ts

```diff
@@ -25,6 +25,18 @@ export interface UserRealm {
   type: string;
 }
 
+export interface ApiKey {
```

Comment:
nit: can we add a JS doc describing what this interface represents? This would take us back to having all APIs 100% documented.

Reply:
Resolved in this commit [731080b](https://github.com/elastic/kibana/pull/204114/commits/731080b08a9c21b4bc7bae66bc8faa6c10defc74)